### PR TITLE
This branch is to create event handling functioning components and update state

### PR DESCRIPTION
**Pull request by steveWDamesJr**
-Create React App
-Add Directories: Src / Components
-Add files: Header.js, InputTodo.js, Navbar.js, TodoContainer.js, TodoItem.js, TodosList.js
-Add linters
-Update Readme
-Update MIT file

-Debug Linters

### CODE REVIEW

File: src/components/TodoContainer.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        id: 3,
+        title: 'Deploy website',
+        completed: false,
+      },
+    ],
+  };
+
+  handleChange = (id) => {
+    console.log('clicked', id);
+  };
```
Comment (deyemiobaa) [inline src/components/TodoContainer.js]: - [x]  Inside your TodoContainer, the handleClick function doesn't up the state of the component, and this causes the checkbox to be inactive even when it is clicked on.

An option is to use the `this.setState()` anytime `handleClick()` is called to update the right object

**Review by deyemiobaa — CHANGES_REQUESTED** [review]
Hi @steveWDamesJr, great job working on the project and getting the components to render on the screen.

There are some issues that you need to work on to make this project complete. check the review at the bottom for more info

To highlight:

- Your project is professional 💯
- Your repo is professional 💯
- Well structured files 💯

Other issues to work on:

- You need to setup linters for this project

File: src/components/TodoContainer.js (hunk 1 above)
Comment (steveWDamesJr) [reply src/components/TodoContainer.js]: Thank you for your kind review and methods to improve!

Comment (steveWDamesJr) [reply src/components/TodoContainer.js]: I have implemented the changes. TodoContainer now updated to allow state change of boolean on check.

image

**Review by deyemiobaa — APPROVED** [review]
Hi @steveWDamesJr 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

**Pull request merged**